GettyImages.Api/WebHelper.cs

```diff
@@ -154,7 +130,8 @@ internal WebHelper(Credentials credentials, string baseAddress, DelegatingHandle
                     _credentials.ResetAccessToken();
                     using (var retryClient = new HttpClient(await GetHandlersAsync(headerParameters)))
                     {
-                        httpResponse = await retryClient.PutAsync(uri, new FormUrlEncodedContent(queryParameters));
```

Comment:
Same deal here, not using `bodyParameter` on the `UnauthorizedException` retry.  See comment above.